CCPAPI/test.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python3
+
+##########################################################################
+#                                                                        #
+#                               CCPAPI                                   #
+#                                                                        #
+#  This program is free software; you can redistribute it and/or modify  #
+#  it under the terms of the GNU Library General Public License as       #
+#  published by the Free Software Foundation; version 2 or later of the  #
+#  License.                                                              #
+#                                                                        #
+#  This program is distributed in the hope that it will be useful,       #
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of        #
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the          #
+#  GNU General Public License for more details.                          #
+#                                                                        #
+#          Copyright 2020 Paul RASCLE www.openfields.fr                  #
+#                                                                        #
+##########################################################################
+
+import sys
+sys.path.append('/home/paul/projets/CloudCompare/install/lib/cloudcompare')
+from PyQt5.QtWidgets import QApplication
+app = QApplication(sys.argv)
+import cloudCompare as cc
+cc.CCLib.ScalarField.initNumpyApi() # to do once before dealing with numpy
+cloud = cc.loadPointCloud("/home/paul/projets/CloudCompare/data/altiXYZ/RGEALTI_FXX_0845_6446_MNT_LAMB93_IGN69.xyz")
```

Comment:
All of the tests with file loading/saving should be modified so whoever builds this can perform the tests. Use paths not specific to your computer and include the specific test data somewhere. Using as small of a data set as possible to perform the test. You can have the file names/locations be set in CMake

Reply:
Yes, you are right, this is in my short term TODO list...

Reply:
I pushed a first step to avoid specific paths, and I generate needed data on my tests. I still need to set some environment manually (path, pythonpath) to launch the tests. To be completed.